src/App.tsx

```diff
@@ -15,49 +17,77 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPreparedGoods(goods: string[],
```

Comment:
```suggestion
enum SortType {
  LENGTH = 'length',
  ALPHABETICALLY = 'alphabetically'
}

function getPreparedGoods(goods: string[],
```

Reply:
you can create an enum

Reply:
https://www.typescriptlang.org/docs/handbook/enums.html
more info you can find here